src/llmperf/common.py

```diff
@@ -12,7 +12,9 @@
 SUPPORTED_APIS = ["openai", "anthropic", "litellm"]
 
 
-def construct_clients(llm_api: str, num_clients: int) -> List[LLMClient]:
+def construct_clients(
+    llm_api: str, num_clients: int, get_token_len: Callable
```

Comment:
`Callable`에 대해서는 인자 형과 반환 형을 적어주면 좋겠습니다:

```suggestion
    llm_api: str, num_clients: int, get_token_len: Callable[[str], int]
```

다른 곳 `Callable`도 비슷하게 수정하는 게 어떨까 싶습니다.

Reply:
반영했습니다 [15da784](https://github.com/furiosa-ai/llmperf/pull/3/commits/15da7844d8e95f9ae30e6812d90ece8f92258500)